ironic_bug_dashboard/__init__.py

```diff
@@ -14,37 +16,26 @@
 template_path = os.path.dirname(os.path.realpath(__file__))
 aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader(template_path))
 
+parser = argparse.ArgumentParser(description='Load configuration from commandline.')
+parser.add_argument('project_name', nargs='?', type=str, help='Name of the project.')
+args, unknown_args = parser.parse_known_args()
+
+LOG.info("*" * 80)
+LOG.info("Configuration options gathered from:")
+LOG.info("command line args: %s", args)
+
+config = simple_lp.load_config(args.project_name)
+if config is None or len(config) <= 0:
+    LOG.error('Configuration file cannot be empty.')
+    sys.exit()
```

Comment:
This will return exit code 0 (success), you probably want to pass something different